code/game/machinery/cloning.dm

```diff
@@ -420,8 +420,6 @@
 		to_chat(occupant, "<span class='notice'><b>There is a bright flash!</b><br><i>You feel like a new being.</i></span>")
 		mob_occupant.flash_act()
 
-	mob_occupant.adjustBrainLoss(mob_occupant.getCloneLoss())

```

Comment:
?

Reply:
That was the previous implementation of organ decay applied to the brain post-cloning. There's a similar line in defib revival that I've cut. No longer necessary since that's actually a game mechanic now.

Reply:
just making sure this wasn't an unlisted change, got it.